e2e-tests/cypress/e2e/Dev-tools.cy.ts

```diff
@@ -16,7 +16,7 @@ describe('Dev tools', () => {
 
     cy.log('should verify GET /_index_template successful with 403 status');
     DevTools.sendRequest('GET /_index_template/');
-    DevTools.verifyIf403Status();
+    DevTools.verifyIf200Status();
```

Comment:
wow, this looks strange. Why do we have 200 right now? Maybe, ES not pass sth it should not pass?

Reply:
No, it was a problem with the test assertion it remembered the previous call result, that's why it was 403 (but it should be 200). Now we are waiting  for the request to start here https://github.com/beshu-tech/readonlyrest-e2e-tests/pull/15/files#diff-78852126a0872b210632d24b016a1bddb6cdc70a6296b35ae7ceac9150f33742R38